WriteVHDLSyntax.py

```diff
@@ -317,6 +319,7 @@ def writeMemoryUtil(memDict, memInfoDict):
                 if "MPAR" in mtypeB:
                     tName = "t_"+mtypeB+"_ADDR"
                     ss += "  subtype "+tName+" is std_logic_vector("+str(8+memInfo.bxbitwidth)+" downto 0);\n" 
+                    #ss += "  subtype "+tName+" is std_logic_vector("+str(7+memInfo.bxbitwidth)+" downto 0);\n" 
```

Comment:
@aryd Can this line be removed if it's not used?:
```suggestion
```

Reply:
Removed.